### PR TITLE
WIP: Gapminder Specific Analysis

### PR DESCRIPTION
**Pull request by AnthonyOfSeattle**
This version of the git lesson follow repository creation and maintenance on the folder produced by the R for reproducible data science lesson. It goes through adding all files and subdirectories, and substitutes modifying the README.md file for the mars file. Most of the lesson is left intact, and I only changed what I really had to.

### CODE REVIEW

**Comment by AnthonyOfSeattle** [comment]
As a prototype, I have just edited episodes 3 & 4 to get a feel for whether this is a good way to go about the lesson. Not much has to change, which is nice, and its not too much work to run through the lesson and copy and paste the results of commands.

I think I left exercises untouched for now. May change or prune later.

Thoughts?

File: _episodes/03-create.md [hunk 1]
```diff
@@ -65,6 +46,11 @@ $ ls
 ~~~
 {: .language-bash}
 
+~~~
+data  functions  gapminder-analysis.rmd  results
+~~~
+{: .output}
+
 But if we add the `-a` flag to show everything,
 we can see that Git has created a hidden directory within `planets` called `.git`:
```
Comment (drammock) [inline _episodes/03-create.md]:
```suggestion
 we can see that Git has created a hidden directory within `gapminder-analysis` called `.git`:
```

File: _episodes/04-changes.md [hunk 2]
```diff
@@ -52,20 +54,22 @@ $ ls
...
 
 ~~~
-mars.txt
+data  functions  gapminder-analysis.rmd  README.md  results
 ~~~
 {: .output}
 
 
-`mars.txt` contains a single line, which we can see by running:
+`README.md` contains a 2 lines, which we can see by running:
```
Comment (drammock) [inline _episodes/04-changes.md]:
```suggestion
`README.md` contains 2 lines, which we can see by running:
```

File: _episodes/04-changes.md [hunk 3]
```diff
@@ -109,30 +118,38 @@ $ git status
...
+	data/
+	functions/
+	gapminder-analysis.rmd
+	results/
 
 ~~~
 {: .output}
 
-Git now knows that it's supposed to keep track of `mars.txt`,
+Git now knows that it's supposed to keep track of `README.txt`,
```
Comment (drammock) [inline _episodes/04-changes.md]:
```suggestion
Git now knows that it's supposed to keep track of `README.md`,
```

File: _episodes/04-changes.md [hunk 4]
```diff
@@ -194,29 +257,34 @@ and the log message Git was given when the commit was created.
...
+# Gapminder Analysis
+
+**Author:** Anthony Valente
+**Depends:** ggplot2, dplyr
+
+This repository contains analyses of life expectancy vs GDP.
 ~~~
 {: .output}
 
+The "**" notation with make the "Author" and "Depends" keywords bold when the markdown file is rendered in a browser.
```
Comment (drammock) [inline _episodes/04-changes.md]:
```suggestion
The `**` notation will make the "Author" and "Depends" keywords bold when the markdown file is rendered in a browser.
```

**Pull request merged**